deploy/cloudbuild-release.yaml

```diff
@@ -1,110 +1,67 @@
-# This cloudbuild is run on the creation of new tags, which should signify releases.
 timeout: 1800s
 
 steps:
+# Set up builder for muti-arch builds.
+- name: 'gcr.io/cloud-builders/docker'
+  args: ['run', '--privileged', 'linuxkit/binfmt:v0.7']
+  id: 'initialize-qemu'
+- name: 'gcr.io/cloud-builders/docker'
+  args: ['buildx', 'create', '--name', 'mybuilder']
```

Comment:
I think you could use `create --use --platform ${_DOCKER_BUILDX_PLATFORMS}` here to (a) limit the builders used and (b) to skip the `select-builder` step immediately below.